lib/main.js

```diff
@@ -74,10 +74,12 @@ export default {
           const line = Number.parseInt(match[1], 10) - 1;
           const col = Number.parseInt(match[2], 10) - 1;
           messages.push({
-            type: match[3],
-            text: match[4],
-            filePath,
-            range: helpers.generateRange(textEditor, line, col),
+            severity: match[3].toLowerCase(),
```

Comment:
Will `tidy` _always_ return a value that is either `info`, `warning`, or `error`? Any other value is invalid here.

In other providers I solved this issue with something similar to this:
```js
function getSeverity(givenSeverity) {
  const severity = givenSeverity.toLowerCase();
  return ['info', 'warning', 'error'].includes(severity) ? severity : 'warning';
}
```